src/structures/GuildMember.js

```diff
@@ -46,6 +46,12 @@ class GuildMember extends Base {
      */
     this.nickname = null;
 
+    /**
+     * When this member will be unmuted, if they are muted
+     * @type {?Date}
+     */
+    this.communicationDisabledUntilTimestamp = null;
```

Comment:
So... is it a date or a timestamp?
Also this could probably be called communicationDisabledTimestamp, thoughts @discordjs/the-big-4 ?

Reply:
changed it to be a number now

Reply:
Please resolve the conversation then